src/core_plugins/kbn_vislib_vis_types/public/controls/gauge_options.html

```diff
@@ -36,6 +36,15 @@
   </div>
 
   <div class="kuiSideBarFormRow">
+    <label class="kuiSideBarFormRow__label" for="addLegend">
+      Show Legend
+    </label>
+    <div class="kuiSideBarFormRow__control">
+      <input class="kuiCheckBox" id="addLegend" type="checkbox" ng-model="vis.params.addLegend">
```

Comment:
because vis.params.addLegend is read by legend (for every chart with legend ... as legend is actually a separate component)